.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# Problem Statement
+- Problem statement
+
+# Design
+-  For Bug, Describe the fix here.
+-  For Feature, Post the link for design
+
+# Coding
+   Checklist for Author
+-  [ ] Coding conventions are followed and code is consistent
```

Comment:
allign as there is an extra space

Reply:
@mukul-seagate11 It is part of markdown syntax.